backend/src/database/migrations/V1686173729__activityOrganizations.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE public."activities" ADD COLUMN "organizationId" uuid;
+
+ALTER TABLE public."activities" ADD FOREIGN KEY ("organizationId") REFERENCES organizations(id);
+
+create table "memberSegmentAffiliations" (
+    id uuid not null,
+    "memberId" uuid not null,
+    "segmentId" uuid not null,
+    "organizationId" uuid,
+    constraint "memberSegmentAffiliations_pkey" primary key (id),
+    foreign key ("memberId") references members (id) on delete cascade,
+    foreign key ("segmentId") references segments (id) on delete cascade,
+    foreign key ("organizationId") references organizations (id) on delete cascade,
+    unique("memberId", "segmentId")
+);
+
+UPDATE activities
+SET "organizationId" = (
+    SELECT mo."organizationId"
+    FROM "memberOrganizations" AS mo
+    WHERE mo."memberId" = activities."memberId"
+    ORDER BY mo."createdAt" DESC
+    LIMIT 1
+);
```

Comment:
This doesn't look right. Shouldn't we also filter by `tenantId`, and maybe `segmentId` ?

Reply:
tenantId filter is already implicitly in `"memberId"` since each member can belong to only one tenant - 
I didn't include segments because, in the prod data, we don't have segments yet. Tho probably it'll be better to add it for the staging data - I updated the query to also join by organizationSegments :+1: